scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -985,6 +985,7 @@ def process(self, pipeline, done, message, lock, results):
         self.instance.setup_handler(self.env)
 
         if op == "filter":
+            logger.step(f"Applying filters {self.instance.name}")
```

Comment:
in all these places, just the same
```suggestion
            logger.step(f"op:{op} instance:{self.instance.name}")
```
this allows to grep/parse/filter logs much easier than a bunch of new and different entries.